pkgs/development/libraries/grpc/default.nix

```diff
@@ -3,13 +3,13 @@
 }:
 
 stdenv.mkDerivation rec {
-  version = "1.36.1"; # N.B: if you change this, change pythonPackages.grpcio-tools to a matching version too
+  version = "1.36.4"; # N.B: if you change this, change pythonPackages.grpcio-tools to a matching version too
```

Comment:
Did you check pythonPackages.grpcio-tools?

Reply:
yes, they didn't publish new releases https://pypi.org/project/grpcio-tools/